src/controllers/controller.polarArea.js

```diff
@@ -129,6 +129,7 @@ module.exports = DatasetController.extend({
 		me._updateRadius();
 
 		meta.count = me.countVisibleElements();
+		start -= Math.PI / 2.0; // radialLinear scale expects 0 to be at top
```

Comment:
this comment isn't enough explanation for me still :-)  is there some other piece of code that interprets 0 as being elsewhere than the top?